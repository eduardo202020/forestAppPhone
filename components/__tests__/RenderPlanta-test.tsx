import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

function RenderPlanta() {
  return <Text>Test component funcionando</Text>;
}

// Mock de react-native-gesture-handler
test('TestComponent se renderiza sin errores', () => {
  const tree = renderer.create(<RenderPlanta />).toJSON();
  expect(tree).toMatchSnapshot();
});
