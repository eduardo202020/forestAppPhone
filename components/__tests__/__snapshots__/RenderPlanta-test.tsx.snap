// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent se renderiza sin errores 1`] = `
<Text>
  Test component funcionando
</Text>
`;
